.travis.yml

```diff
@@ -14,3 +14,7 @@ env:
 script: bundle exec ./go ci_build
 notifications:
   webhooks: http://project-monitor.cf.18f.us/projects/hub/status
+
+script: bundle exec rake run_about_yml_check
```

Comment:
I'm pretty indifferent. Happy to edit that PR to remove the link, or leave
it in and leave this branch alive.

On Tue, Dec 29, 2015 at 11:33 AM, Mike Bland <notifications@github.com>
wrote:

> In .travis.yml <https://github.com/18F/hub/pull/503#discussion_r48551815>:
>
> > @@ -14,3 +14,7 @@ env:
> >  script: bundle exec ./go ci_build
> >  notifications:
> >    webhooks: http://project-monitor.cf.18f.us/projects/hub/status
> > +
> > +script: bundle exec rake run_about_yml_check
>
> I'd think the 18F/about_yml#71 <https://github.com/18F/about_yml/pull/71>
> is sufficient without needing to link to a commit, but you can just keep
> this original commit in the branch history and it'll stay alive.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/hub/pull/503/files#r48551815>.
>